tests/suites/test_suite_rsa.function

```diff
@@ -16,6 +16,9 @@ void rsa_invalid_param( )
     mbedtls_rsa_context ctx;
     const int invalid_padding = 42;
     const int invalid_hash_id = 0xff;
+    const mbedtls_md_type_t md_alg_none = MBEDTLS_MD_NONE;
```

Comment:
This should be removed as its no longer used.

Reply:
Thanks for pointing it out.

Reply:
Fixed it